pandas/io/pytables.py

```diff
@@ -3021,6 +3026,15 @@ def write_array(self, key, value, items=None):
                 elif is_timedelta64_dtype(value.dtype):
                     self._handle.create_array(self.group, key, value.view("i8"))
                     getattr(self.group, key)._v_attrs.value_type = "timedelta64"
+                elif is_sparse(value):
+                    # TODO: think about EA API for this.
+                    # value._write_hdf5(self)
```

Comment:
is `value._write_hd5(self)` a commented-out line or part of the TODO comment above?

Reply:
Part of the TODO.